tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/test_sms_report.py

```diff
@@ -0,0 +1,77 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import Base
+
+from gaiatest import GaiaTestCase
+import time
+from gaiatest.apps.messages.app import Messages
+
+
+class TestSmsReport(GaiaTestCase):
+
+    def test_sms_report(self):
+        """This test sends a text message to itself. It waits for a reply message.
+           Then it checks the info in the report.
+        """
+        # Now here, then these locators will be in Report object
+        _text_type = (By.CSS_SELECTOR, 'p[data-l10n-id="message-type-sms"]')
+        _sent_time = (By.CSS_SELECTOR, 'span.sent-timestamp')
+        _received_time = (By.CSS_SELECTOR, 'span.received-timestamp')
+        _sender = (By.CSS_SELECTOR, 'a[data-name]')
+        
+        # Configuring the test
+        _text_message_content = "Automated Test %s" % str(time.time())
+        # _my_muber this way only during the local tests
+        _my_number = "+34644967894"
```

Comment:
We parameterize this data in the variable called `self.testvars['phone_number']`. `self.testvars` is the content of the file called `testvars.json`, and 'phone_number' is the key in that JSON